samples/hello_world/src/main.c

```diff
@@ -47,9 +50,9 @@ void Flash_Test(const struct device *flash, uint32_t FLASH_ADDR, uint8_t EVEN_OD
 		}
 		if(errorcode == -1) {
 			printf("%s\nFlash erase at 0x%08x did not produce correct results%s\n", ATTR_ERR, FLASH_ADDR,ATTR_RST);
-			for(uint8_t i = 0; i < FLASH_RW_SIZE; i++) {
-				printf("%s 0x%02x%s", ATTR_INF, flash_data_read[i],i%FORMATTER==0?"\n":"");			
-			} printf("\n");
+			// for(uint8_t i = 0; i < FLASH_RW_SIZE; i++) {
+			// 	printf("%s 0x%02x%s", ATTR_INF, flash_data_read[i],i%FORMATTER==0?"\n":"");			
+			// } printf("\n");
 		} else {
 			printf("%s\nSuccessfully performed erase to flash with code:%d%s\n", ATTR_INF, errorcode,ATTR_RST);
 		}
```

Comment:
LInes 51 to 58 do not seem to make sense. They have already been covered in the if errorcode < 0 in line 41 above

Reply:
This is actually serving a different purpose. After erasing the flash, if we don't see all 0xFF values, then it means the flash erase did not go well. The error check at line 41 suggests an overall protocol error. This is where I had found earlier that the erase command supported by our flash was not part of the DTS SFDP Header bytes. Then I modified that header to make it correct.